llama-index-core/llama_index/core/async_utils.py

```diff
@@ -42,10 +42,6 @@ def asyncio_run(coro: Coroutine) -> Any:
                 "Or, use async entry methods like `aquery()`, `aretriever`, `achat`, etc."
             )
 
-    except Exception as e:
-        # Catch any other exceptions and re-raise with more context
-        raise type(e)(f"Error running coroutine: {e!s}") from e
```

Comment:
ooc: could we achieve same affect if we just raise without argument?

```python
except Exception:
    raise
```

Reply:
Yea, but that seems kind of redundant no? Instead now, we just let the error raise as is

Reply:
Yeah I think redundant for sure. 😊